launchable/commands/subset.py

```diff
@@ -338,6 +339,8 @@ def run(self):
                 _output, _rests = output, rests
                 if is_output_exclusion_rules:
                     _output, _rests = _rests, _output
+                elif is_observation:
```

Comment:
[question]
Is it correct that you can only get change for only one of `is_output_exclusion_rules` and `is_observation`?
I mean, even if `is_output_exclusion_rules == True` and `is_observation == True`, you will not go through `elif is_observation` clause since it is `elif` instead of `if`.

Reply:
Oh, nice catch. Yes, the user will be able to use both options in one request. So, I'll fix it